crates/aranya-client/tests/common/mod.rs

```diff
@@ -207,6 +215,50 @@ impl TeamCtx {
 
         Ok(())
     }
+
+    pub async fn create_and_add_team(&mut self) -> Result<TeamId> {
+        // Create the initial team, and get our TeamId and PSK.
+        let cfg = TeamConfig::builder().build()?;
+        let response = {
+            self.owner
+                .client
+                .create_team(cfg.clone())
+                .await
+                .expect("expected to create team")
+        };
+        info!(?response.team_id);
```

Comment:
It's possible to extract the fields directly from the returned response type like this:
```suggestion
        let CreateTeamResponse {team_id: team_id, seed: seed} = self.owner
                .client
                .create_team(cfg.clone())
                .await
                .expect("expected to create team");
        info!(?team_id);
```